src/api/app/controllers/webui/project_controller.rb

```diff
@@ -95,6 +95,8 @@ def subprojects
   def new
     @project = Project.new
     @project.name = params[:name] if params[:name]
+
+    @show_restore_message = params[:restore_option] && Project.deleted?(params[:name])
```

Comment:
The restore is placed in the GET parameter in order to show this hint: https://github.com/openSUSE/open-build-service/pull/3271/commits/283a60311d3c663e1b14f57b3682aa5708878ccd#diff-b0495897e6c3c6e1c9c2aba47a0afc40R16